aries_cloudagent/protocols/issue_credential/v2_0/formats/ld_proof/handler.py

```diff
@@ -270,10 +272,17 @@ async def _get_suite_for_detail(
         )
 
         did_info = await self._did_info_for_did(issuer_id)
-        verification_method = verification_method or self._get_verification_method(
-            issuer_id
+        verkey_id_strategy = self.profile.context.inject(DefaultVerificationKeyStrategy)
```

Comment:
Shouldn't this be `Base` instead of Default, as you can override the default with a custom one? It would't make sense to make the injection token the default implementation IMO.

Reply:
So this should be renamed to `BaseVerificationKeyStrategy`? This would mean the base (abstract) class would be named `VerificationKeyStrategyBase` and the default implementation `DefaultVerificationKeyStrategy`, is this correct?

Reply:
@yvgny that is correct